src/views/Home/Home.tsx

```diff
@@ -81,32 +80,29 @@ const Home: React.FC = () => {
   const TranslateString = useI18n()
 
   return (
-    <>
-      <LeadInBanner />
```

Comment:
Some fairly agressive diffs in here. In each instance the `<LeadInBanner>` is being removed as well as the wrapping fragment